lms/djangoapps/email_marketing/migrations/0007_auto_20170803_0903.py

```diff
@@ -0,0 +1,29 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('email_marketing', '0006_auto_20170711_0615'),
+    ]
+
+    operations = [
+        migrations.RenameField(
```

Comment:
You really can't do this.
The code currently running reads this field and your new code expects the new name, but it isn't deployed yet.

This type of code change is incompatible with red/black blue/green deploys